test/test_core.py

```diff
@@ -2860,6 +2860,12 @@ def test_pthread_stdout_after_main(self):
     # streams were locked when the main thread returned.
     self.do_runf(test_file('pthread/test_pthread_stdout_after_main.c'))
 
+  @node_pthreads
+  def test_pthread_proxy_to_pthread(self):
+    self.set_setting('PROXY_TO_PTHREAD')
+    self.set_setting('EXIT_RUNTIME')
+    self.do_run_in_out_file_test(test_file('pthread/test_pthread_proxy_to_pthread.c'))
```

Comment:
`test_pthread_proxy_to_pthread.c` looks like it already existed but this is a new test. is there another test that also uses it?

Reply:
yes, its an existing test that runs as a browser test.  Many of these pthread tests we also run in the browser test suite.